src/ChainSafe.Gaming.SygmaClient/Config.cs

```diff
@@ -1,33 +1,53 @@
-﻿using ChainSafe.Gaming.SygmaClient.Types;
+﻿using System.Collections.Generic;
+using ChainSafe.Gaming.SygmaClient.Dto;
+using ChainSafe.Gaming.SygmaClient.Types;
+using ChainSafe.Gaming.Web3.Environment;
 
 namespace ChainSafe.Gaming.SygmaClient
 {
     public class Config
     {
-        public Config(uint chainId, Environment environment)
+        private IHttpClient httpClient;
+
+        private uint chainId;
+
+        private Environment environment;
+
+        public Config(IHttpClient http, uint chain)
```

Comment:
Think this could've gone with IProjectConfig because project config has the chain ID